client/.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
```

Comment:
I'm not sure I understand `"node": true` here. The client code should run only in the browser, right?

Reply:
I wasn't sure whether it's necessary since the client code should run in the browser, but Node does need to be installed to run the frontend. From a quick search now it doesn't seem to be necessary, I'll remove it.